tests/unit/rpc/handlers/LedgerEntryTests.cpp

```diff
@@ -2193,6 +2195,76 @@ generateTestValuesForParametersTest()
             .expectedError = "malformedRequest",
             .expectedErrorMessage = "Malformed request.",
         },
+        ParamTestCaseBundle{
+            .testName = "InvalidVault_Type",
+            .testJson =
+                R"JSON({
+                    "vault": 0
+                })JSON",
+            .expectedError = "malformedRequest",
+            .expectedErrorMessage = "Malformed request.",
+        },
+        ParamTestCaseBundle{
+            .testName = "InvalidVault_NotHex",
+            .testJson =
+                R"JSON({
+                    "vault": "invalid_hex"
+                })JSON",
+            .expectedError = "malformedRequest",
+            .expectedErrorMessage = "Malformed request.",
+        },
+        ParamTestCaseBundle{
+            .testName = "MissingOwner",
+            .testJson =
+                R"JSON({
+                    "vault": { "seq": 1 }
+                })JSON",
+            .expectedError = "malformedRequest",
+            .expectedErrorMessage = "Malformed request.",
+        },
+
+        ParamTestCaseBundle{
+            .testName = "MissingSeq",
+            .testJson =
+                R"JSON({
+                    "vault": { "owner": "abcd" }
+                })JSON",
+            .expectedError = "malformedRequest",
+            .expectedErrorMessage = "Malformed request.",
+        },
+        ParamTestCaseBundle{
+            .testName = "SeqNotInteger",
+            .testJson =
+                R"JSON({
+                 "vault": {
+                    "owner": "abcd",
+                    "seq": "notAnInteger"
+                }})JSON",
+            .expectedError = "malformedRequest",
+            .expectedErrorMessage = "Malformed request.",
+        },
+        ParamTestCaseBundle{
+            .testName = "InvalidOwnerFormat",
+            .testJson =
+                R"JSON({
+                "vault" : {
```

Comment:
```suggestion
                "vault": {
```

And in similar places